tests/manage/test_pvresize_with_node_down.py

```diff
@@ -0,0 +1,228 @@
+"""
+OCS-368
+"""
+import logging
+import pytest
+import json
+from time import sleep
+import threading
+
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import tier1, ManageTest
+from ocs_ci.ocs.resources.pvc import PVC
+from ocs_ci.ocs.resources.pod import Pod
```

Comment:
import multiple modules using a single line:

```python
from ocs_ci.ocs.resources.pvc import Pod, get_ceph_tools_pod
```

Reply:
@ebenahar `ocs_ci.ocs.resources.pod` not `pvc` but yes these should be done on one line.

Reply:
correct

Reply:
Done.